packages/devextreme/js/__internal/grids/new/grid_core/options_validation/controller.ts

```diff
@@ -0,0 +1,22 @@
+import { isDefined } from '@js/core/utils/type';
+import errors from '@js/ui/widget/ui.errors';
+import { DataController } from '@ts/grids/new/grid_core/data_controller';
```

Comment:
QUnit fails because we forget to set ```index``` here
(Unfortunately, QUnit cannot load ```index``` automatically without mention)

```suggestion
import { DataController } from '@ts/grids/new/grid_core/data_controller/index';
```

Reply:
done